frontend/src/APIClients/BaseAPIClient.ts

```diff
@@ -0,0 +1,66 @@
+import axios, { AxiosResponse, AxiosRequestConfig } from "axios";
+import { camelizeKeys, decamelizeKeys } from "humps";
+import jwt from "jsonwebtoken";
+
+import USER_ACCESS_TOKEN_KEY from "../constants/AuthConstants";
+
+const baseAPIClient = axios.create({
```

Comment:
Creating some axios interceptors on a base API client to run some logic that should be executed for every request made or response received
https://github.com/axios/axios#interceptors